testing/unittests/jlibtests2.cpp

```diff
@@ -347,11 +347,11 @@ attribute: bytesRead = 16
             CPPUNIT_ASSERT(recorder.startRecording("all=true", "eventtrace.evt", false));
             CPPUNIT_ASSERT(recorder.isRecording());
             CPPUNIT_ASSERT(recorder.stopRecording(nullptr));
-            std::stringstream out;
+            StringBuffer out;
             Owned<IEventVisitor> visitor = createVisitor(out);
             CPPUNIT_ASSERT(visitor.get());
             CPPUNIT_ASSERT(readEvents("eventtrace.evt", *visitor));
-            CPPUNIT_ASSERT_EQUAL(std::string(expect), out.str());
+            CPPUNIT_ASSERT_EQUAL(std::string(expect), std::string(out.str()));
```

Comment:
info: CPPUNIT_ASSERT_EQUAL_STR() is a useful macro to use in this situation:

`CPPUNIT_ASSERT_EQUAL_STR(expect, out.str());`